plugins/textpattern.md

```diff
@@ -101,8 +102,11 @@ The block patterns do not have an `end` property. This allows for patterns to be
 
 ```js
 tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
-  plugin: 'textpattern lists',
+  selector: 'textarea', // change this value according to your HTML
+  // make sure to have the required plugins here,
```

Comment:
Could this be dropped back to being 2 lines now?

Reply:
Done